powerline/segments/common/sys.py

```diff
@@ -13,7 +13,8 @@
 cpu_count = None
 
 
-def system_load(pl, format='{avg:.1f}', threshold_good=1, threshold_bad=2, track_cpu_count=False):
+def system_load(pl, format='{avg:.1f}', threshold_good=1, threshold_bad=2,
+		track_cpu_count=False, short=False):
```

Comment:
hmm... I just found that this code is using tab for indentation.
So, will using all spaces for this line be suitable? Or to keep this consistency in same file, should i leave it without new line?